sdcm/cluster.py

```diff
@@ -1306,15 +1308,29 @@ def install_mgmt(self, scylla_repo, scylla_mgmt_repo):
         if not (self.is_rhel_like() or self.is_debian() or self.is_ubuntu()):
             raise ValueError('Unsupported Distribution type: {}'.format(str(self.distro)))
         if self.is_debian8():
+            self.remoter.run(cmd="sudo sed -i -e 's/jessie-updates/stable-updates/g' /etc/apt/sources.list")
+            self.remoter.run(cmd="sudo touch /etc/apt/sources.list.d/jessie-backports.list")
+            self.remoter.run(
+                cmd="sudo echo 'deb http://archive.debian.org/debian jessie-backports main' | sudo tee /etc/apt/sources.list.d/jessie-backports.list")
+            self.remoter.run(cmd="sudo touch /etc/apt/apt.conf.d/99jessie-backports")
+            self.remoter.run(
+                cmd="sudo echo 'Acquire::Check-Valid-Until \"false\";' | sudo tee /etc/apt/apt.conf.d/99jessie-backports")
+            self.remoter.run('sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys F76221572C52609D', retry=3)
             install_transport_https = dedent("""
-                if [ ! -f /etc/apt/sources.list.d/backports.list ]; then sudo echo 'deb http://http.debian.net/debian jessie-backports main' | sudo tee /etc/apt/sources.list.d/backports.list > /dev/null; fi
-                apt-get install apt-transport-https
-                apt-get install gnupg-curl
-                apt-get update
-                apt-key adv --fetch-keys https://download.opensuse.org/repositories/home:/scylladb:/scylla-3rdparty-jessie/Debian_8.0/Release.key
-                apt-get install -t jessie-backports ca-certificates-java -y
-                        """)
+                            if [ ! -f /etc/apt/sources.list.d/backports.list ]; then sudo echo 'deb http://http.debian.net/debian jessie-backports main' | sudo tee /etc/apt/sources.list.d/backports.list > /dev/null; fi
+                            apt-get install apt-transport-https
+                            apt-get install gnupg-curl
+            """)
             self.remoter.run('sudo bash -cxe "%s"' % install_transport_https)
+            self.remoter.run(cmd="sudo apt-get update", ignore_status=True)
+
+
+            install_transport_backports = dedent("""
+                apt-key adv --fetch-keys https://download.opensuse.org/repositories/home:/scylladb:/scylla-3rdparty-jessie/Debian_8.0/Release.key
+                # apt-get install -t jessie-backports ca-certificates-java -y
+                apt-get install ca-certificates-java -y
+            """)
+            self.remoter.run('sudo bash -cxe "%s"' % install_transport_backports)
 
         if self.is_debian9():
```

Comment:
@yarongilor 

reported a new issue found in review this PR: 
- https://github.com/scylladb/scylla-cluster-tests/issues/883